x-pack/plugins/secops/public/components/timeline/properties/translations.ts

```diff
@@ -51,6 +51,10 @@ export const HISTORY = i18n.translate('xpack.secops.timeline.properties.history'
   defaultMessage: 'History',
 });
 
+export const IS_VIEWING = i18n.translate('xpack.secops.timeline.properties.isViewing', {
```

Comment:
Same note about ending the `i18n` keys with a suffix per the guidelines it it's not too much trouble.